tests/bach_tests.ts

```diff
@@ -64,17 +64,17 @@ function minuet1(options: TestOptions): void {
   system.addConnector('singleRight');
   system.addConnector('singleLeft');
 
-  id('m1a').addModifier(0, f.Fingering({ number: '5' }));
+  id('m1a').addModifier(f.Fingering({ number: '5' }), 0);
```

Comment:
0s not required in this file. I will no longer comment on this topic on the next files.